addons/website_links/static/src/js/website_links_charts.js

```diff
@@ -305,7 +305,12 @@ sAnimations.registry.websiteLinksCharts = sAnimations.Class.extend({
     _onGraphTabClick: function (ev) {
         ev.preventDefault();
         $('.graph-tabs li a').tab('show');
-        _.chain(this.charts).pluck('chart').invoke('update'); // Force NVD3 to redraw the chart
```

Comment:
Strange... this line did not work in 12.0 so ?

Reply:
It was working in 12.0
Somehow it is not working anymore on saas-12.1:

![screenshot from 2019-03-01 11-57-20](https://user-images.githubusercontent.com/30048408/53633967-3a365900-3c19-11e9-8328-656150631536.png)

The weird part is that on the `start` method, there is a
```javascript
nv.utils.windowResize(function () {
    _.chain(self.charts).pluck('chart').invoke('update');
});
```
That works well, if you resize your windows the charts are correctly redrawn..
But on this `_onGraphTabClick` the same line is not working anymore, even in a setTimeout.

Anyway I googled a bit and found these solutions:
```javascript
window.dispatchEvent(new Event('resize'));
```
or
```javascript
$(window).trigger('resize');
```
None is working, except the first if wrapped into a `setTimeout(0)`..